sensei/CatalystAnalysisAdaptor.h

```diff
@@ -43,11 +43,12 @@ class SENSEI_EXPORT CatalystAnalysisAdaptor : public AnalysisAdaptor
   /** Adds a pipeline defined in a Catalyst python script. The Catalyst Python
    * script can be automatically generated using the ParaView GUI on a
    * representative dataset. The sensei::VTKPosthocIO and sensei::VTKAmrWriter
-   * can be used to obtain such data.
+   * can be used to obtain such data. If ParaView is unable to determine the
+   * version of the Script, if will default to the given version.
    */
-  virtual void AddPythonScriptPipeline(const std::string &fileName,
+  virtual void AddPythonScriptPipeline(const std::string& fileName,
     const std::string& resultProducer, const std::string& steerableSourceType,
-    const std::string& resultMesh);
+    const std::string& resultMesh, int scriptVersion = 1);
```

Comment:
since all scripts generated in paraview since version 5.9 will be version 2, I recommend making the default `scriptVersion = 2` instead. Hopefully if people are using legacy scripts, they can turn to this explicitly.

Reply:
IIRC, the default version will only be 2 if you build your ParaView with Catalyst 2, which we did not know about at first. But I agree that the default script version should be 2 nonetheless!